tests/test_io_elementwise.py

```diff
@@ -47,6 +54,39 @@ def store(request, tmp_path) -> H5Group | ZarrGroup:
             file.close()
 
 
+sparse_formats = ["csr", "csc"]
+SIZE = 1000
+
+
+@pytest.fixture(scope="function", params=sparse_formats)
+def sparse_format(request):
+    return request.param
+
+
+def create_dense_store(store):
+    X = np.random.randn(SIZE, SIZE)
+
+    write_elem(store, "X", X)
+    return store
+
+
+def create_sparse_store(sparse_format, store):
+    import dask.array as da
+
+    X = sparse.random(
+        SIZE,
+        SIZE,
+        format=sparse_format,
+        density=0.01,
+        random_state=np.random.default_rng(),
+    )
+    X_dask = da.from_array(X, chunks=(100, 100))
+
+    write_elem(store, "X", X)
+    write_elem(store, "X_dask", X_dask)
+    return store
```

Comment:
If we're going to use these in more places, I would suggest not making them square. This has definitely caused problems for us in the past. Maybe even make the shape/ chunk size a parameter?

I would also write a short doc string about what is actually being returned. I think it's a little strange that one of these groups has two things in it, while other only has one.

Reply:
I think we can just do the chunking along the axis.  The shape shouldn't matter.  We can reoslve it from `sparse_format`